### PR TITLE
Fix pdf links

### PR DESCRIPTION
**Pull request by start974**
Fixing some links to the Coq-Doc.
Changed the links of the exercises to the branch master.

### CODE REVIEW

File: doc/chapter-powers.tex [hunk 1]
```diff
@@ -155,7 +155,7 @@ \subsection{A logarithmic exponentiation  function}
 \vspace{4pt}
 
 \emph{From Module
-\href{../theories/html/hydras.additions.FirstSteps.html}{additions.FirstSteps}}
+\href{../theories/html/FirstSteps.html}{additions.FirstSteps}}
```
Comment (Zimmi48) [inline doc/chapter-powers.tex:158]: Everywhere else, you kept the `additions.` prefix, but not here. Why?

File: doc/part-hydras.tex [hunk 2]
```diff
@@ -2263,7 +2263,7 @@ \subsection{A class for ordinal notations}
 \end{Coqsrc}
 
 \vspace{4pt}
-\noindent\emph{From Library~\href{../theories/html/hydras.OrdinalNotations.Definitions.html}{OrdinalNotations.Definitions}}
+\noindent\emph{From Library~\href{../theories/html/hydras.OrdinalNotations.ON_mult.html}{OrdinalNotations.ON\_Generic}}
```
Comment (Zimmi48) [inline doc/part-hydras.tex:2266]: Is it a mistake that it `ON_mult` in the link but `ON_Generic` in the text?

File: doc/part-hydras.tex [hunk 3]
```diff
@@ -4701,7 +4702,8 @@ \subsubsection{Using a stronger inductive predicate.}
 \end{Coqsrc}
 
 The following lemma is an application of the strict inequality 
-\showmath{\alpha < \omega ^\alpha}. If \showmath{\alpha} is strongly accessible, then, by definition, \showmath{\omega^\alpha} is accessible, thus \showmath{\alpha} is \emph{a fortiori} accessible.
+$\alpha < \omega ^\alpha.$ If \alpha is strongly accessible, then, by definition,
+$\omega^\alpha$ is accessible, thus \alpha is \emph{a fortiori} accessible.
```
Comment (Zimmi48) [inline doc/part-hydras.tex:4706]: Did you forget some `$` here?

File: doc/chapter-powers.tex (hunk 1 above)
Comment (start974) [reply doc/chapter-powers.tex:158]: It's a mistake, there are need `additions.`

**Pull request merged**